pkg/controllers/jobset_controller.go

```diff
@@ -277,29 +277,16 @@ func (r *JobSetReconciler) executeFailurePolicy(ctx context.Context, js *jobset.
 		return r.failJobSet(ctx, js)
 	}
 
-	// Handle different types of failure policy targets.
-	switch js.Spec.FailurePolicy.Operator {
-	case jobset.TerminationPolicyTargetAny:
-		// To reach this point a job must have failed, and TerminationPolicyTargetAny applies to any job
-		// in the JobSet, so we can skip directly to executing the restart policy.
-		return r.executeRestartPolicy(ctx, js, ownedJobs)
-	default:
-		return fmt.Errorf("invalid termination policy target %s", js.Spec.FailurePolicy.Operator)
-	}
+	// To reach this point a job must have failed, and TerminationPolicyTargetAny applies to any job
```

Comment:
Nit: given `TerminationPolicyTargetAny` is removed here, this comment might be confusing, maybe consider rephrasing?

Reply:
good catch, done.